application-mocca-calendar-ui/src/main/resources/MoccaCalendar/MoccaCalendarMacroTranslations.de.xml

```diff
@@ -39,24 +39,24 @@
   <minorEdit>false</minorEdit>
   <syntaxId>xwiki/2.1</syntaxId>
   <hidden>true</hidden>
-  <content>rendering.macro.moccacalendar.name=Kalendar
+  <content>rendering.macro.moccacalendar.name=Kalender
 rendering.macro.moccacalendar.description=Zeigt einen Kalender an
 rendering.macro.moccacalendar.parameter.filter.name=Zeige Termine von ...
-rendering.macro.moccacalendar.parameter.filter.description=Mögliche Werte\: 'wiki','space' oder 'page', wobei 'page' nur Termine eines Dokuments anzeigt, 'space' alle Termine des aktuellen Bereichs anzeigt, und 'wiki' alle Termine im Wiki anzeigt.
-rendering.macro.moccacalendar.parameter.calendarDoc.name=Kalenderdokument
-rendering.macro.moccacalendar.parameter.calendarDoc.description=Das Kalender-Dokument, dessen Termine angezeigt werden (wird nur verwendet, wenn im obigen Wert zur Anzeige der Termine 'page' verwendet wird. Falls leer, wird das aktuelle Dokument verwendet. Wenn das Dokument kein Kalender ist, werden keine Termine angezeigt.
+rendering.macro.moccacalendar.parameter.filter.description=Mögliche Werte: 'wiki','space' oder 'page', wobei 'page' nur Termine eines Kalenders anzeigt, 'space' alle Termine in Unterseiten anzeigt, und 'wiki' alle Termine im Wiki anzeigt.
+rendering.macro.moccacalendar.parameter.calendarDoc.name=Kalenderseite
+rendering.macro.moccacalendar.parameter.calendarDoc.description=Die Kalender-Seite, deren Termine angezeigt werden (wird nur verwendet, wenn im obigen Wert zur Anzeige der Termine 'page' verwendet wird. Falls leer, wird die aktuelle Seite verwendet. Wenn ddie Seite kein Kalender ist, werden keine Termine angezeigt.
```

Comment:
Ouch, typo: `Wenn ddie Seite ...`

Reply:
I let you edit the translation on weblate, not merging right now :)

Reply:
It should be fixed by now on weblate. :)

Reply:
FTR: I keep it like that for now, still trying to understand why weblate doesn't commit automatically.

Reply:
Ok. Is weblate expected to augment the existing pull request, or should it create a new one?  
Anyway, as I do not know how to help with debugging this I will just wait and see. :)

Reply:
> Is weblate expected to augment the existing pull request

it augments them: basically it push forces them

Reply:
The may the force be with weblate back soon.